REFRAME.md

```diff
@@ -40,20 +40,57 @@ Given that a client C wants to request information from some server S:
 
 ### HTTP + DAG-JSON
 
-All messages MUST be encoded as DAG-JSON and use explicit content type `application/vnd.ipfs.rpc+dag-json; version=1`
+All messages sent in HTTP body MUST be encoded as DAG-JSON and use explicit content type `application/vnd.ipfs.rpc+dag-json; version=1`
 
+Requests MUST be sent as either:

```

Comment:
How does a client know which kind of request it should send? Does it do trial and error per-method to see what it supports (e.g. try GET and if that fails then try POST)?

Reply:
(we added list of cachable Reframe methods (that should default to GET) + rate-limiting of POSTs on HTTP layer (`429 Too Many Requests`, which will suggest to use GET for these highly cachable responses)